app/models/tool_draft.rb

```diff
@@ -35,4 +37,22 @@ def display_short_name
   def display_entry_title
     entry_title || '<Untitled Tool Record>'
   end
+
+  private
+
+  def set_metadata_specification
+    # this is a hidden fieldset added to UMM-T to document the metadata version
+    # and specs that should be autopopulated
+    # TODO: we should try to populate these values from the schema enums
```

Comment:
Do we have a ticket to document this work?

Reply:
No but as discussed today, I think we may want to think about it or investigate a bit. Unless you or @jceaser think we should have a ticket to investigate, then we should write one.

Reply:
lets have a wider talk about this field and the "created by" field then maybe write a ticket.